Automattic/wp-calypso
# Introduce JetpackBenefits component and subcomponents that can be use…

Introduce JetpackBenefits component and subcomponents that can be used to show key features of a Jetpack plan or product

#### Changes proposed in this Pull Request

* This PR adds a `JetpackBenefits` component and related subcomponents to highlight benefits Jetpack provides for a given site ID and product
* This PR is related to / a sub-part of https://github.com/Automattic/wp-calypso/pull/54669 which introduces a new cancellation flow for Jetpack plans and products. The functionality contained in this PR is also contained in #54669

#### Testing instructions

* Check out this branch locally
* Run the automated tests for this branch: `yarn run test-client client/blocks/jetpack-benefits `
* All tests should pass - tests focus on confirming the output of components with high variability. 

The visual output of this component can be seen by testing PR #54669
* Either check out the branch from PR #54669 locally or use the Calypso live link on that branch to get a preview. 
* Review the testing instructions on PR #54669 to check the visual output

![image](https://user-images.githubusercontent.com/5714212/130685847-d80d9796-b87c-41a0-9ae6-f92e6a381296.png)

![image](https://user-images.githubusercontent.com/5714212/130685869-b4e71812-cc1b-4a9c-a444-46d9db08bf9f.png)
